clang/lib/CodeGen/CGDebugInfo.cpp

```diff
@@ -3834,12 +3839,14 @@ llvm::DIType *CGDebugInfo::CreateTypeNode(QualType Ty, llvm::DIFile *Unit) {
 
   case Type::TemplateSpecialization:
     return CreateType(cast<TemplateSpecializationType>(Ty), Unit);
+  case Type::HLSLAttributedResource: {
+    return CreateType(cast<HLSLAttributedResourceType>(Ty), Unit);
+  }
```

Comment:
nit: remove unnecessary braces.
```suggestion
  case Type::HLSLAttributedResource:
    return CreateType(cast<HLSLAttributedResourceType>(Ty), Unit);
```